### PR TITLE
[WIP] include docs from AbstractAlgebra

### PR DESCRIPTION
**Pull request by rfourquet**
This simply copies the doc files from AA and include them while building Oscar docs.
Of course, this would also need a few words to explain to the reader how this is orgnized.
If this is the desired direction, docs from other cornerstones can be added in the same way.

@fieker did you have in mind something like this?

### CODE REVIEW

**Comment by fingolfin** [comment]
Note: BinaryBuilder.jl somehow includes docs from BinaryBuilderBase.jl, see e.g. https://juliapackaging.github.io/BinaryBuilder.jl/dev/reference/#BinaryBuilderBase.get_concrete_platform-Tuple{AbstractPlatform} -- I haven't checked how they do it, but might be worth having a look?

File: docs/make.jl [hunk 1]
```diff
@@ -1,18 +1,78 @@
 using Documenter, Oscar, DocumenterMarkdown
+using AbstractAlgebra
+
+AA_dir = pkgdir(AbstractAlgebra)
+Oscar_dir = pkgdir(Oscar)
+
+cp(joinpath(AA_dir, "docs", "src"), joinpath(Oscar_dir, "docs", "src", "AAsrc"), force=true)
```
Comment (fingolfin) [inline docs/make.jl:7]: just for my understanding: would a symlink perhaps also work?

**Comment by fieker** [comment]
If this works, then yes, this is a BIG step forward. Can I see the new documentation?

**Comment by rfourquet** [comment]
> Note: BinaryBuilder.jl somehow includes docs from BinaryBuilderBase.jl, 

As far as I understand, they just pull docstrings from BinaryBuilderBase into the docs of BinaryBuilder, but no documentation in markdown files is imported.

File: docs/make.jl (hunk 1 above)
Comment (rfourquet) [reply docs/make.jl:7]: > just for my understanding: would a symlink perhaps also work?

I'm pretty sure yes, but I never bothered to understand how symlinks work on Windows, and the docstring of `symlink` says:

> This function raises an error under operating systems that do not support soft symbolic links, such as Windows XP.

Given how little work it is to copy the files here, I didn't think twice, but am happy to use `symlink`, or even `symlink` in `try-catch` followed by a copy, if you think it's worthwhile.

**Comment by rfourquet** [comment]
> Can I see the new documentation?

I just pushed a new commit with `push_preview = true`, I never used it but let's see if it works: https://oscar-system.github.io/Oscar.jl/previews/PR186

Otherwise, I don't know of a way, besides building the docs locally by pulling this PR: ` git fetch origin   pull/186/head:pr186; git checkout pr186`

**Comment by rfourquet** [comment]
Ok, as I made this PR from a fork, preview won't work, according to Documenter's note:

> Pull requests made from forks will not have previews. Hosting previews requires access to the deploy key. Therefore, previews are available only for pull requests that were submitted directly from the main repository.

**Comment by thofma** [comment]
I have added `DOCUMENTER_KEY` as a secret.

**Comment by thofma** [comment]
Sorry, wrong repository :)

**Pull request closed without merging**